business_rules/operators.py

```diff
@@ -96,7 +96,7 @@ def non_empty(self):
         return bool(self.value)
 
     @type_operator(FIELD_TEXT)
-    def contains_in(self, other_string):
+    def is_in(self, other_string):
```

Comment:
You really need a substring option? Or are you intending this to be about membership in a list of things?

Reply:
substring option